assets/js/buttons-examples.js

```diff
@@ -216,6 +216,13 @@ $(document).ready(function () {
           ],
         },
 
+        {
+          type: "setting",
+          label: "Setting Button",
+          classes: "setting-btn",
+          text: "",
+        },
+
```

Comment:
Please arrange it in alphabetical order.

Reply:
Done!!